### DISCUSSION TITLE
zk Syncs Bridge

### DISCUSSION THREAD

**Discussion by talhazullfiqar** [Cyfrin/foundry-full-course-cu#2555] [Q&A]
hey im trying to send my sepolia testnet funds to zksync sepolia testnet by zksync bridge i have 0.15 sepoliaETH in my wallet but when i enter ammount ( 0.025 sepholiaETH ) to transfer to my zksync network it says Insufficient balance  also charging fee of: 
0.174923303142948198 ETH
![Screenshot 2024-09-26 193240](https://github.com/user-attachments/assets/b70212c6-bcce-4a47-9a58-c567933cad90)

**Comment by EngrPips** [comment]
Hello @talhazullfiqar, I will advise you to get more sepolia eth and retry. Chainlink Faucet https://faucets.chain.link/sepolia is a good faucet you can try.

**Reply by talhazullfiqar** [reply]
I transferred ETH from both the Chainlink and Google Cloud faucets, which is why I have a total of 0.15 ETH on Sepolia. However, the problem is that I can't get more because they have set the limits to only 0.1 and 0.05 ETH. One thing I don't understand is why zkSync is charging a fee for the testnet.

**Reply by EngrPips** [reply]
You can try to get more tomorrow. Testnet is the closest thing to real blockchain, so they emulate everything on the real blockchain, which includes charging a gas fee for any transaction sent.

**Comment by Hepcat93** [comment]
Oh, seemingly we're at the same level of progress :D I've just did the transfer myself. Initially I got my SepoliaETH from the Google faucet. I requested for 0.05 SepoliaETH two times within one day break, so in total I recevied 0.1 Sepolia ETH. I transferred 0.025 SepoliaETH to zkSync. So I had 0.0608 SepoliaETH left in my wallet. And now the breakdown: approximately 0.02726 SepoliaETH was exactly bridged to zkSync and 0.01192 was taken as a fee in the Sepolia Etherium test network. So I got 0.0272 ETH in zkSync out of which about 0.0006 ETH was deducted as a free. And in the very very end I got 0.0266 in zkSync wallet. That happened due to the partial fee refund which in its turn seemingly happened to partial network load at the time. If I paid full fee I would end with the intended 0.025 SepoliaETH in zkSync.

Hopefully this information will be useful!

**Reply by EngrPips** [reply]
Thanks for the heads-up, Friend.
